examples/typescript/ucloud/package.json

```diff
@@ -27,6 +27,7 @@
     "@types/node": "^14.0.26",
     "cdktf-cli": "0.0.0",
     "jest": "27.1.0",
+    "ts-node": "10.4.0",
```

Comment:
why this specific version?

Reply:
Because it's the latest version, using the latest tag was a bit too risky for me

Reply:
I'm wondering if this will ever be updated when it's fixed. It might also suggest to users that there's a reason why it's a fixed version, even though it's just there since it happened to be the current version at some point.

Reply:
Would adding a ^ solve your concern?

Reply:
Yes, I think that would be good 👍

Reply:
👍